drivers/sensor/adi/adxl345/adxl345.c

```diff
@@ -54,60 +54,239 @@ static int adxl345_reg_access_spi(const struct device *dev, uint8_t cmd, uint8_t
 		.count = 2,
 	};
 
+	int ret = 2;
```

Comment:
Why assign to 2?

Reply:
That was leftover from debugging, and is removed now